csharp-mode.el

```diff
@@ -432,13 +446,12 @@ casts and declarations are fontified.  Used on level 2 and higher."
 	      "\\("
 	      (concat "\\sw\\|\\s \\|[=\n\r*.:]\\|"
 		      "\"[^\"]*\"\\|'[^']*'")
-	      "\\)*>")
+	      "\\)*/?>")
```

Comment:
Assuming this is to support the syntax for nullable types? If so, might be worth a tiny comment to document just that.

Reply:
In fact, it isn't. This is in the codedoc to match an optional `/>` for the self closing tag.

```xml
<Tag>content</Tag>
<Tag />
```

Reply:
Oh yeah, when I see it in a greater context (not just the diff) that becomes immediately obvious. My bad.